config/settings.yml.example

```diff
@@ -3,7 +3,12 @@
 :database:
 
 # URL of the smart proxy, used for reporting back
-:callback_url: 'https://localhost:8443'
+:callback_url: 'http://localhost:9292'
```

Comment:
It was my default for running proxy locally. Anyways, thinking more about the ssl authentication and how the callback works, I've at the end changed the behavior in a way, where the core doesn't send the callback through the proxy, but rather calls foreman directly: there is no need to go though proxy for this part + if we did, we would need to add the  proxy to it's trusted hosts so that the call is accepted.